### DISCUSSION TITLE
TreeSelect Scroll to expanded

### DISCUSSION THREAD

**Discussion by ifflanb** [ant-design/ant-design#28310] [Q&A]
Hi, I am using the Tree Select component to display locations and am setting the treeDefaultExpandedKeys to the default location for the user.
This works but I would like the tree select to scroll to the default location when they open the tree select rather than having to manually scroll to it. Is this possible?

thanks

**Comment by xrkffgg** [comment]
TreeNode has no ID, otherwise can ues scrollIntoView
